src/map/CustomGridLayer.tsx

```diff
@@ -83,7 +83,7 @@ export default function CustomGridLayer() {
         const latLines = lats.map((lat, index) => {
             return (
                 <Polyline
-                    key={lat + index}
+                    key={`lat-${lat}-${index}`}
```

Comment:
Nice - I assume this fixes the grid line weirdness?

Reply:
I'm hoping so. I wasn't seeing the grid weirdness but I was getting warnings for these and I think it would make sense.